DEVELOPING.md

```diff
@@ -22,8 +22,11 @@ We discuss implementation issues on [amphtml-discuss@googlegroups.com](https://g
 
 ### Installation
 
-1. `npm i`
-2. `edit /etc/hosts` and map `ads.localhost` and `iframe.localhost` to `127.0.0.1`:
+1. Install [NodeJS](https://nodejs.org).
+2. In the repo directory and run `npm i` command to install the required npm packages.
```

Comment:
I think you can remove 'and' here, or put something like 'Go into the repo directory' :smile: